private/protocol/restjson/build_bench_test.go

```diff
@@ -9,9 +9,7 @@ import (
 	"testing"
```

Comment:
Shouldn't these files of been updated in the generated marshaler change?

Reply:
Yea, this is a rebase issue.